metadata/OCAPI/shopOCAPI.json

```diff
@@ -23,7 +23,7 @@
             },
             {
                 "resource_id": "/orders",
-                "methods": ["post", "get"],
+                "methods": ["post"],
```

Comment:
The ticket says to do something else `"Proposed Resolution: We asked the merchant to change “methods”: [“post”, “get”] to “methods”: [“get”], for “resource_id”: “/orders/*“,`

Reply:
That one is already with `GET` only

Reply:
Oh, it was wrong on the ticket, good catch @asecaida